woocommerce/woocommerce
# [Product Block Editor]: tweak styles and layout of AdviceCard component

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

On this PR:

* Set the height of the AdviceCard according to the _Products in this group_ section
<img width="682" alt="Screenshot 2024-01-04 at 08 02 29" src="https://github.com/woocommerce/woocommerce/assets/77539/9d73d661-ba1e-4cf5-abdd-8db69ad59ca4">

* Don't use Flex components set, and replace them by regular div elements
* Adjust the component layout






Closes # .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Enable the Product Editor app
2. Enable the Linked Product flag
  - Enable the WCA plugin
  - Click on `Tools` -> `WCA Test Helper`
  - Click on `Features` tab
  - Toggle on the `product-linked` feature 

<img width="456" alt="Screenshot 2023-12-26 at 11 36 05" src="https://github.com/woocommerce/woocommerce/assets/77539/a065ce77-fdbc-45bd-99dc-0c00e5cad328">

3. Create a new product by using the new Product Editor app
4. Go to the Linked Product tab
5. Check the height and layout in general of the Advice card instance

<img width="682" alt="image" src="https://github.com/woocommerce/woocommerce/assets/77539/f8a43f40-5c05-406d-8fb3-c45543219a28">

7. 


### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
